docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,574 @@
+# Text
+
+There are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.
+
+<a name="textLabel"></a>
+## TextLabel
+
+The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents` namespace contains the class.
+
+The `TextLabel` class is lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display a text using a text label, see [NUI Hello World Tutorial](../../get-started/nui/quickstart.md).
+
+**Figure: Text label positioned to top left**
+
+![Text label example, positioned to top left](media/TextLabelTopLeft.png)
+
+<a name="create"></a>
+### Create Text Label
+
+To create a text label:
+
+1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+
+    ```csharp
+    Window window = Window.Instance;
+    TextLabel label = new TextLabel("Hello World");
+    window.Add(label);
+    ```
+
+    You can also create the `Tizen.NUI.BaseComponents.TextLabel` class instance separately and define the label text by setting its `Text` property:
+
+    ```csharp
+    TextLabel label = new TextLabel();
+    label.Text = "Hello World";
+    ```
+
+    > [!CAUTION]
+    > To display the label properly, the `Text` property must be a UTF-8 string. Any `CR+LF` new line characters are replaced by `LF`.
+
+2.  Define the label position on-screen with the `ParentOrigin` property of the `Tizen.NUI.BaseComponents.TextLabel` class:
+
+    ```csharp
+    TextLabel label = new TextLabel("Hello World");
+    label.ParentOrigin = ParentOrigin.TopLeft;
+    ```
+
+
+<a name="font"></a>
+### Select Fonts

```

Comment:
\### Set Fonts of TextLabel

Reply:
@mijin85cho How about?

### Set Font Styles for TextLabel

Reply:
@nawab-samsung

For the TextLabel : As like TextField or TextEditor, I think TextLabel is better than Text Label.
For font : I think it is acceptable.

Reply:
@mijin85cho thanks. 

@JoogabYun 
Request to change to

### Set Font of TextLabel